docs/eventing/broker/rabbitmq-broker/README.md

```diff
@@ -40,21 +41,21 @@ To use the RabbitMQ Broker, you must have the following installed:
 
     1. Create a YAML file using the following template:
 
-        ```bash
-        apiVersion: rabbitmq.com/v1beta1
-        kind: RabbitmqCluster
-        metadata:
-          name: <cluster-name>
-        ```
-        Where `<cluster-name>` is the name you want for your RabbitMQ cluster,
-        for example, `hello-world`.
```

Comment:
Putting the spaces back breaks the YAML colouring.

Before:
![image](https://user-images.githubusercontent.com/90614196/144097740-99183fc5-e5fb-4b5f-8873-42a91ae39c48.png)

After:
![image](https://user-images.githubusercontent.com/90614196/144097753-0fa29da1-cc63-4d9e-a288-4d0edcb7eb44.png)

Do you think that we could remove the nested list entirely @snneji, so no `1. & 1.` inside the ordered list?

Reply:
I just noticed that the problem is with the code language on line 43. It should say `yaml` not `bash`. The colours should work with the nested list if it has the right language.

Reply:
That's what line 43 used to be, but I've since fixed it to `yaml`.

I have reverted the indentation & the second ordered list to exactly what it used to be. YAML syntax highlighting is broken as shown above, but I want to leave it out of the scope of this PR.